switching between the wallets tab and overview tab continuously caused the app to crash

```
panic: runtime error: index out of range [3] with length 3

goroutine 182 [running]:
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).txStakingSection.func1.3.1.1.2({{{0x0, 0x0}, {0x27a, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:950 +0x30f
gioui.org/layout.(*List).Layout(0xc00564ce88, {{{0x0, 0x0}, {0x27a, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/list.go:122 +0x17c
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).txStakingSection.func1.3.1.1({{{0x0, 0x0}, {0x27a, 0xf41f4}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:948 +0xef
gioui.org/layout.Flex.Layout({0x85?, 0x1?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x27a, 0xf41f4}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x27a, 0xf41f4}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:88 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x27a, 0xf41f4}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x0, 0x0}, {0x27a, 0xf41f4}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:87 +0xe5
gioui.org/layout.Inset.Layout({0x6d02ef?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x27a, 0xf41f4}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x0, 0x0}, {0x298, 0xf4212}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:85 +0x109
github.com/crypto-power/cryptopower/ui/cryptomaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x8, 0x8, 0x8, 0x8}, 0x0}, {{{0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/border.go:21 +0x98
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2({{{0x0, 0x0}, {0x298, 0xf4212}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:83 +0x1d6
gioui.org/layout.Stack.Layout({0x0?}, {{{0x0, 0x0}, {0x298, 0xf4212}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x298, 0xf4212}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:46 +0x2aa
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x298, 0xf4212}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:105 +0x238
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x298, 0xf4212}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x298, 0xf422e}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x298, 0xf422e}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout({0xffffffffffffffff, 0xffffffffffffffff, 0x1, {0xff, 0xff, 0xff, 0xff}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:42 +0xe5
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout2(...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:37
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).pageContentWrapper.func1({{{0x298, 0x0}, {0x298, 0xf422e}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:1166 +0x22a
gioui.org/layout.Flex.Layout({0x40?, 0xb?, 0xbb?, 0xc0?}, {{{0x298, 0x0}, {0x298, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).pageContentWrapper(0xc00564c000, {{{0x298, 0x0}, {0x298, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:1144 +0x3f8
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).txStakingSection.func1.3.1({{{0x298, 0x0}, {0x298, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:940 +0x170
gioui.org/layout.Inset.Layout({0x7bb0b40?, 0xc0?, 0x17d?, 0x0?}, {{{0x298, 0x0}, {0x298, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).txStakingSection.func1.3({{{0x2a2, 0x0}, {0x2a2, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:939 +0x89
gioui.org/layout.Flex.Layout({0xbe?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:134 +0x738
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).txStakingSection.func1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:937 +0x3f8
gioui.org/layout.Flex.Layout({0x7b?, 0x1?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:88 +0x85
gioui.org/layout.Direction.Layout(0x8, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x544, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:87 +0xe5
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x544, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x544, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:85 +0x109
github.com/crypto-power/cryptopower/ui/cryptomaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x544, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/border.go:21 +0x98
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2({{{0x544, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:83 +0x1d6
gioui.org/layout.Stack.Layout({0x0?}, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:46 +0x2aa
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:105 +0x238
gioui.org/layout.Inset.Layout({0x1ff?, 0x0?, 0x544?, 0x0?}, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x8, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xffffffffffffffff, 0x0, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:42 +0xe5
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).txStakingSection(0xc00564c000, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:908 +0x20a
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).layoutDesktop.func1.1.1.1.1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:299 +0x55
gioui.org/layout.(*List).Layout(0xc00564c028, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/list.go:122 +0x17c
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).layoutDesktop.func1.1.1.1({{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:298 +0xa5
gioui.org/layout.Inset.Layout({0x9a?, 0x0?, 0x32d8?, 0x0?}, {{{0x0, 0x0}, {0x544, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).layoutDesktop.func1.1.1({{{0x0, 0x0}, {0x546, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:297 +0xc9
gioui.org/layout.Direction.Layout(0x8, {{{0x0, 0x0}, {0x546, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).layoutDesktop.func1.1({{{0x546, 0x0}, {0x546, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:296 +0xb0
gioui.org/layout.(*List).Layout(0xc00a8d6e50, {{{0x546, 0x0}, {0x546, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/list.go:122 +0x17c
gioui.org/widget/material.ListStyle.Layout({0xc00a8d6e00, {0xc00a8d6e00, {0x40000000, 0x40000000, {0x0, 0x0, 0x0, 0x0}}, {0x42180000, 0x40c00000, ...}}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/widget/material/list.go:290 +0x20c
github.com/crypto-power/cryptopower/ui/cryptomaterial.ListStyle.Layout(...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/list_scroll.go:39
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).layoutDesktop.func1({{{0x0, 0x37c}, {0x550, 0x37c}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:295 +0x205
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x37c}, {0x550, 0x37c}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/components.UniformPadding({{{0x0, 0x3ac}, {0x780, 0x3ac}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/components/components.go:100 +0x197
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).layoutDesktop(0x0?, {{{0x0, 0x3ac}, {0x780, 0x3ac}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:294 +0x1c5
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).Layout(0xc00564c000, {{{0x0, 0x3ac}, {0x780, 0x3ac}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/overview_page.go:282 +0x105
gioui.org/layout.Flex.Layout({0x5?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:134 +0x738
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:88 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x780, 0x417}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:87 +0xe5
gioui.org/layout.Inset.Layout({0x7c35b68?, 0xc0?, 0x6b3f37?, 0x0?}, {{{0x780, 0x417}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x780, 0x417}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:85 +0x109
github.com/crypto-power/cryptopower/ui/cryptomaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x780, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/border.go:21 +0x98
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2({{{0x780, 0x417}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:83 +0x1d6
gioui.org/layout.Stack.Layout({0x0?}, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:46 +0x2aa
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:105 +0x238
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x576cf3?, 0x0?}, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xfffffffffffffffe, 0x1, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:42 +0xe5
github.com/crypto-power/cryptopower/ui/page/root.(*HomePage).layoutDesktop.func1({{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/home_page.go:373 +0x24b
gioui.org/layout.Stack.Layout({0x52?}, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:78 +0x498
github.com/crypto-power/cryptopower/ui/page/root.(*HomePage).layoutDesktop(0xc00018e000, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/home_page.go:367 +0x12b
github.com/crypto-power/cryptopower/ui/page/root.(*HomePage).Layout(0xc00018e000, {{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/home_page.go:363 +0x105
github.com/crypto-power/cryptopower/ui.(*Window).prepareToDisplayUI.func2({{{0x0, 0x0}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, 0x783c15f3f5, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/window.go:288 +0x122
gioui.org/layout.Stack.Layout({0x40?}, {{{0x780, 0x417}, {0x780, 0x417}}, {0x3f800000, 0x3f800000}, {0x2007c40, 0xc0043680c8}, {0xc14b120f6234f6ea, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui.(*Window).prepareToDisplayUI(0xc0042f41e0, {{0xc14b120f6234f6ea, 0x783c15f3f5, 0x2f2b700}, {0x3f800000, 0x3f800000}, {0x780, 0x417}, {0x0, 0x0, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/window.go:303 +0x40d
github.com/crypto-power/cryptopower/ui.(*Window).handleFrameEvent(0xc0042f41e0, {{0xc14b120f6234f6ea, 0x783c15f3f5, 0x2f2b700}, {0x3f800000, 0x3f800000}, {0x780, 0x417}, {0x0, 0x0, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/window.go:241 +0x225
github.com/crypto-power/cryptopower/ui.(*Window).HandleEvents(0xc0042f41e0)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/window.go:206 +0x2b9
main.main.func3()
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/main.go:110 +0x1d
created by main.main
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/main.go:108 +0x61b
exit status 2
```

--- a/ui/page/root/overview_page.go
+++ b/ui/page/root/overview_page.go
@@ -1331,7 +1331,7 @@ func (pg *OverviewPage) ratesRefreshComponent() func(gtx C) D {
 }
 
 func (pg *OverviewPage) loadTransactions() {
-	pg.transactions = make([]*multiWalletTx, 0)
+	transactions := make([]*multiWalletTx, 0)
 	wal := pg.WL.AllSortedWalletList()
 	for _, w := range wal {
 		txs, err := w.GetTransactionsRaw(0, 3, libutils.TxFilterAllTx, true)
@@ -1341,17 +1341,18 @@ func (pg *OverviewPage) loadTransactions() {
 		}
 
 		for _, tx := range txs {
-			pg.transactions = append(pg.transactions, &multiWalletTx{tx, w.GetWalletID()})
+			transactions = append(transactions, &multiWalletTx{tx, w.GetWalletID()})
 		}
 	}
 
-	sort.Slice(pg.transactions, func(i, j int) bool {
-		return pg.transactions[i].Timestamp > pg.transactions[j].Timestamp
+	sort.Slice(transactions, func(i, j int) bool {
+		return transactions[i].Timestamp > transactions[j].Timestamp
 	})
 
-	if len(pg.transactions) > 3 {
-		pg.transactions = pg.transactions[:3]
+	if len(transactions) > 3 {
+		transactions = transactions[:3]
 	}
+	pg.transactions = transactions
 
 	pg.loadStakes()
 }

